azure-pipelines-official.yml

```diff
@@ -37,6 +38,30 @@ stages:
     - powershell: Write-Host "##vso[task.setvariable variable=SourceBranchName]$('$(Build.SourceBranch)'.Substring('refs/heads/'.Length))"
       displayName: Setting SourceBranchName variable
 
+    - powershell: Write-Host "##vso[task.setvariable variable=IsPrValidationBuild]($(PrNumber) -ne '') -and ($(PrNumber) -ne 'default')"
+      displayName: Setting IsPrValidationBuild variable
```

Comment:
>Pr [](start = 29, length = 2)

nit: `Pr` casing is odd. The usual guidelines is both capital for two letter abbrev. `IsPRValidationBuild`, `PRNumber`

Reply:
Done.